### PR TITLE
Make links to target="_blank" safe

### PR DESCRIPTION
**Pull request by alxndrsn**
Issue: medic/medic-projects#3071

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -548,6 +555,7 @@ module.exports = function(grunt) {
 
   grunt.registerTask('unit', 'Lint and unit tests', [
     'jshint',
+    'exec:blankLinkCheck',
```
Comment (garethbowen) [inline Gruntfile.js]: Our other custom lint rules are in the `precommit` task which is run before CI and can be run as a commit hook. Move this there too.

File: static/css/bootstrap/tests/css-tests.html [hunk 2]
```diff
@@ -1360,8 +1360,8 @@ <h4>Error</h4>
...
       <div class="container">
         <p class="pull-right"><a href="#">Back to top</a></p>
-        <p>Designed and built with all the love in the world <a href="http://twitter.com/twitter" target="_blank">@twitter</a> by <a href="http://twitter.com/mdo" target="_blank">@mdo</a> and <a href="http://twitter.com/fat" target="_blank">@fat</a>.</p>
-        <p>Code licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0" target="_blank">Apache License v2.0</a>. Documentation licensed under <a href="http://creativecommons.org/licenses/by/3.0/">CC BY 3.0</a>.</p>
+        <p>Designed and built with all the love in the world <a href="http://twitter.com/twitter" target="_blank" rel="noopener noreferrer">@twitter</a> by <a href="http://twitter.com/mdo" target="_blank" rel="noopener noreferrer">@mdo</a> and <a href="http://twitter.com/fat" target="_blank" rel="noopener noreferrer">@fat</a>.</p>
+        <p>Code licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0" target="_blank" rel="noopener noreferrer">Apache License v2.0</a>. Documentation licensed under <a href="http://creativecommons.org/licenses/by/3.0/">CC BY 3.0</a>.</p>
```
Comment (garethbowen) [inline static/css/bootstrap/tests/css-tests.html]: Let's not modify bootstrap code. I'm not even quite sure why we have committed this instead of using the version in `node_modules`. Revert this change, then either fix it so bootstrap is included from the node module, _or_ add an exception for bootstrap to your rule and raise an issue to fix the include.

File: templates/partials/reports_list.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
   <p ng-show="error" class="alert alert-danger" role="alert">
     <span ng-show="errorSyntax">
       <span translate>invalid.query</span>
-      <a ui-sref="help.search" target="_blank" translate>Help</a>.
+      <a ui-sref="help.search" target="_blank" rel="noopener noreferrer" translate>Help</a>.
```
Comment (garethbowen) [inline templates/partials/reports_list.html]: We don't have a route for `help.search` any more so I think this is broken. I guess just delete this line altogether?

File: .jshintrc [hunk 4]
```diff
@@ -37,8 +37,7 @@
     "boss"          : false,     // true: Tolerate assignments where comparisons would be expected
     "debug"         : false,     // true: Allow debugger statements e.g. browser breakpoints.
     "eqnull"        : false,     // true: Tolerate use of `== null`
-    "es5"           : false,     // true: Allow ES5 syntax (ex: getters and setters)
-    "esnext"        : false,     // true: Allow ES.next (ES6) syntax (ex: `const`)
+    "esversion"     : 6,
```
Comment (garethbowen) [inline .jshintrc]: Is this related? You will also need to add a `.jshint` with es5 to `static/js` because we can't support es6 in browsers yet.

File: translations/messages-en.properties [hunk 5]
```diff
@@ -891,7 +891,7 @@ help.search.description = <h4>Terms</h4> <p> A query is broken up into terms and
...
...by wrapping the field name in braces, eg\:</p> <pre>Thank you \\{\\{contact.name\\}\\}, \\{\\{patient_name\\}\\} has been registered.</pre> <p><a href\="http\://mustache.github.io/mustache.5.html" target\="_blank" rel\="noopener noreferrer">More information</a>.</p> <h4>Available Fields</h4> <dl class\="horizontal"> <dt><span class\="pre">contact</span></dt> <dd>The contact at the clinic. Use contact.name or contact.phone.</dd> <dt><span class\="pre">clinic</span></dt> <dd>The clinic</dd> <dt><span class\="pre">parent</span></dt> <dd>The parent facility of the clinic</dd> <dt><span class\="pre">grandparent</span></dt> <dd>The grandparent facility of the clinic</dd> <dt><span class\="pre">health_center</span></dt> <dd>The health center</dd> <dt><span class\="pre">district</span></dt> <dd>The district</dd> <dt></dt> <dd>All other fields on the document are included</dd> </dl> <h4>Formatters</h4> <p>To format dates using the configured date format\:</p> <pre>Expected due date is \\{\\{\#date\\}\\}\\{\\{expected_date\\}\\}\\{\\{/date\\}\\}</pre> <p>Or the configured datetime format\:</p> <pre>Message received at \\{\\{\#datetime\\}\\}\\{\\{reported_date\\}\\}\\{\\{/datetime\\}\\}</pre>
```
Comment (garethbowen) [inline translations/messages-en.properties]: Changing translations in the file does nothing because [it'll be overwritten next time they get exported](https://github.com/medic/medic-docs/blob/master/development/translations.md#modifying-any-existing-translation-values) from poeditor. Update the translation for all languages using poeditor, then export all to update the committed .properties files before merging this, then revert this file, then rebase this branch...

File: .jshintrc (hunk 4 above)
Comment (alxndrsn) [reply .jshintrc]: Yeah - es6 should be allowed in `Gruntfile`.

**Comment by alxndrsn** [comment]
All fixed except for POEditor.
